instrumentation/http-client/src/OpenTelemetry/Instrumentation/HttpClient/Raw.hs

```diff
@@ -107,27 +138,52 @@ instrumentResponse
   -> Response a
   -> m ()
 instrumentResponse conf ctxt resp = do
-  tp <- httpTracerProvider
-  ctxt' <- extract (getTracerProviderPropagators $ getTracerTracerProvider tp) (responseHeaders resp) ctxt
+  tracer <- httpTracerProvider
+  ctxt' <- extract (getTracerProviderPropagators $ getTracerTracerProvider tracer) (responseHeaders resp) ctxt
   _ <- attachContext ctxt'
   forM_ (lookupSpan ctxt') $ \s -> do
     when (statusCode (responseStatus resp) >= 400) $ do
       setStatus s (Error "")
-    addAttributes
-      s
-      [ ("http.status_code", toAttribute $ statusCode $ responseStatus resp)
-      -- TODO
-      -- , ("http.request_content_length",	_)
-      -- , ("http.request_content_length_uncompressed",	_)
-      -- , ("http.response_content_length", _)
-      -- , ("http.response_content_length_uncompressed", _)
-      -- , ("net.transport")
-      -- , ("net.peer.name")
-      -- , ("net.peer.ip")
```

Comment:
`net.peer.ip` is deleted because it is superseded by `server.address` which can either contain the contents of `net.peer.name` or `net.peer.ip`.